remoting/README.md

```diff
@@ -0,0 +1,305 @@
+ 
+## Deployment
+
+### Compile, Test & Package
+
+```console
+mvn package
+```
+
+This will generate 
+- a [standalone jar](target/remoting-agent-1.9.5-SNAPSHOT.jar) containing all necessary rdf4j components to build your own repository server.
+- a [pluging jar](target/original-remoting-agent-1.9.5-SNAPSHOT.jar) which maybe dropped into an rdf4j server for remoting support.
+
+### Run Locally
+
+The standalone jar](target/remoting-agent-1.9.5-SNAPSHOT.jar) contains an example application that runs a sample repository against a sample source
+
+```console
+java -jar target/remoting-agent-1.9.5-SNAPSHOT.jar -Dorg.slf4j.simpleLogger.defaultLogLevel=DEBUG
+```
+
+### Containerizing
+
+You could either call
+
+```console
+mvn install -Pwith-docker-image
+```
+
+or invoke the following docker command after a successful package run
+
+```console
+docker build -t ghcr.io/eclipse-tractusx/knowledge-agents/remoting-agent:1.9.5-SNAPSHOT -f src/main/docker/Dockerfile .
+```
+
+This will create a docker image including an extended rdf4j-server as well as an interactive rdf4j-workbench.
+
+To run the docker image, you could invoke this command
+
+```console
+docker run -p 8081:8081 \
+  -v $(pwd)/src/test:/var/rdf4j/config \
+  ghcr.io/eclipse-tractusx/knowledge-agents/remoting-agent:1.9.5-SNAPSHOT
+````
+
+Afterwards, you should be able to access the [local SparQL endpoint](http://localhost:8081/) via
+the browser or by directly invoking a query
+
+```console
+curl --location --request POST 'http://localhost:8081/resources' \
+--header 'Content-Type: application/sparql-query' \
+--header 'Accept: application/json' \
+--data-raw 'PREFIX : <http://example.org/voc#>
+
+SELECT ?x
+WHERE {
+   ?x a :Professor .
+}'
+```
+
+You may manipulate any of the following environment variables to configure the image behaviour.
+Note that there is no builtin security (ssl/auth) for the exposed endpoints.
+This must be provided by hiding them in an appropriate service network layer.
+
+| ENVIRONMENT VARIABLE        | Required  | Example                                                                | Description                          | List |
+|---	                        |---	      |---	                                                                   |---                                   | ---  |
+| JAVA_TOOL_OPTIONS           |           | -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:8090   | JMV (Debugging option)               | X    | 
+
+### Helm Chart
+
+A helm chart for deploying the remoting agent can be found under [this folder](../charts/remoting-agent).
+
+It can be added to your umbrella chart.yaml by the following snippet
+
+```console
+dependencies:
+  - name: remoting-agent
+    repository: https://catenax-ng.github.io/product-knowledge/infrastructure
```

Comment:
After the initial contribution you should have GH workflows in place which will release your application and other artifacts like Helm chart.

Do not link to catenax-ng repositories/GH Org.

Reply:
switched to eclipse tractusx helm repo (dev)